portfolio/src/main/webapp/index.html

```diff
@@ -3,10 +3,14 @@
   <head>
     <meta charset="UTF-8">
     <title>Ayo's Portfolio</title>
+    <script src="https://maps.googleapis.com/maps/api/js?key=AIzaSyCV5SxRGBpxXlDKjKmNVuav4Xpsq0prZjo"></script>
     <link rel="stylesheet" href="style.css">
     <script src="script.js"></script>
   </head>
-  <body onload = "getMessage()">
+  <script async defer
```

Comment:
Hmm, it the problem here that the url parameter for "callback" is initMap, but your actually map creation callback is createMap?

Reply:
The URL has since been deleted, even changing it to initMap didn't work when I tried.